src/slash-bedrock/share/brl-fetch/distros/fedora

```diff
@@ -18,7 +18,14 @@ check_supported() {
 }
 
 speed_test_url() {
-	echo "releases/${target_release}/Everything/${distro_arch}/os/repodata/repomd.xml"
+
+	if [[ $(list_development | tr -d '[:space:]') == *${target_release}* ]]; then
+		url_type="development"
+	else
+		url_type="releases"
+	fi
```

Comment:
Some of Bedrock's shell script runs in environments where bash-isms like the double brackets work, but some doesn't.  A uniform style everywhere is preferred, which means avoiding bash-isms where we can.

Something like this may work:

```
if list_development | sed -e 's/^/ /' -e 's/$/ /' | grep -qF "^${target_release}$"
```

or perhaps this:

```
if ! list_development | awk -v"x=${target_release}" '$0 == x {exit 1}'
```

I haven't tested those - they may be buggy.  More proposed sources of inspiration than firm counter-proposals.  You're welcome to play with it if you have other ideas.

Reply:
The second one seems to work, and that is a clever solution. I like it :+1: 
Just running through a full test now before committing that